features/step_definitions/welsh-language-support.rb

```diff
@@ -1,5 +1,5 @@
 When(/^I have selected the Welsh translation$/) do
-  click_button('in Welsh (Cymraeg)')
+  click_button('Cymraeg')
```

Comment:
@woodingd When I run this test it says "Unable to find button" and the test runner shows a 404: Not found error. Do they work for you?